test/utils/wabt_engine.cpp

```diff
@@ -71,7 +71,13 @@ std::optional<WasmEngine::FuncRef> WabtEngine::find_function(std::string_view na
 WasmEngine::Result WabtEngine::execute(
     WasmEngine::FuncRef func_ref, const std::vector<uint64_t>& args)
 {
-    wabt::interp::Executor executor{&m_env};
+    // WABT Executor/Thread will init both value and call stack vectors with zeros up front.
+    // This is not very smart design. To mitigate the overhead, we set sizes to much lower values
+    // than default ones.
```

Comment:
Can you mention the default sizes?

Also this is the size of the preallocated stack or the limit of the stack?

Reply:
> Also this is the size of the preallocated stack or the limit of the stack?

I'm guessing it's both.